src/style.css

```diff
@@ -0,0 +1,19 @@
+.block1 {
+  background-color: red;
+}
+
+.block2 {
+  background-color: green;
+}
+
+.block3 {
+  background-color: blue;
+}
+
+div {
+  height: 100px;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice